### PR TITLE
reorder steps #2 and #3 in binary search tree project

### PR DESCRIPTION
**Pull request by Drew-Goddyn**
### What are you doing?
 
 Reordered step 2 and 3 to have the student build the class before the instance method.

### CODE REVIEW

**Comment by Drew-Goddyn** [comment]
Looks like layout was fixed in another PR

**Comment by Drew-Goddyn** [comment]
repurposing to just fix the order of steps #1 and #2

**Comment by CouchofTomato** [comment]
Hey @Drew-Goddyn 

Sorry it took so long to deal with this. When you switched the methods it looks like you removed a word so now it reads "write a #buildtree" rather than "write a #buildtree method". Are you interested in fixing this still.

File: ruby_programming/computer_science/project_data_structures_algorithms.md [hunk 1]
```diff
@@ -11,15 +11,19 @@ You'll build a simple binary search tree in this assignment. In this lesson, our
 
   1. Build a `Node` class.  It is should have attributes for the data it stores as well as its left and right children. As a bonus, try including the `Comparable` module and make nodes compare using their data attribute.
 
-  2. Write a method `#build_tree` which takes an array of data (e.g. [1, 7, 4, 23, 8, 9, 4, 3, 5, 7, 9, 67, 6345, 324]) and turns it into a balanced binary tree full of `Node` objects appropriately placed (don't forget to sort and remove duplicates!). The `#build_tree` method should return the level-1 root node.
+  2. Build a `Tree` class which accepts an array when initialized. The `Tree` class should have a `root` attribute which uses the return value of `#build_tree` which you'll write next.
 
-  2. Build a `Tree` class which accepts an array when initialized. The `Tree` class should have a `root` attribute which uses the return value of `#build_tree`.
+  3. Write a `#build_tree` which takes an array of data (e.g. [1, 7, 4, 23, 8, 9, 4, 3, 5, 7, 9, 67, 6345, 324]) and turns it into a balanced binary tree full of `Node` objects appropriately placed (don't forget to sort and remove duplicates!). The `#build_tree` method should return the level-1 root node.
```
Comment (Drew-Goddyn) [inline ruby_programming/computer_science/project_data_structures_algorithms.md]:
```suggestion
  3. Write a `#build_tree` method which takes an array of data (e.g. [1, 7, 4, 23, 8, 9, 4, 3, 5, 7, 9, 67, 6345, 324]) and turns it into a balanced binary tree full of `Node` objects appropriately placed (don't forget to sort and remove duplicates!). The `#build_tree` method should return the level-1 root node.
```

**Comment by Drew-Goddyn** [comment]
@CouchofTomato done!

**Comment by mindovermiles262** [comment]
@Drew-Goddyn it looks like there's a merge conflict in this file now. Can you please fix this?

Thanks

**Pull request merged**